bucketupload/README.md

```diff
@@ -1,3 +1,86 @@
 # bucketupload
 
-Work in progress.
+Uploads a local file to a configured [gocloud.dev/blob] bucket. Allows setting
+the object key with the `Key` parameter, using the filename if not set. The
+`BufferSize` parameter changes the default size in bytes of the chunks that
+will be uploaded in a single request. If 0, the driver will choose a reasonable
```

Comment:
"If `Buffersize` is 0, the driver will chose a reasonable default. `Buffersize` may be ignored by some drivers."